src/site/content/en/handbook/style/index.md

```diff
@@ -0,0 +1,158 @@
+
+{% Compare 'worse', 'Don’t' %}
+> We've added _[a cool thing]_ in DevTools.
+
+This sentence emphasizes the work the developers of DevTools did rather than what it lets the reader do.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Use _[a cool thing]_ in DevTools to _[do neat stuff]_.
+
+This sentence better focuses on what how readers can meet their goals.
+{% endCompare %}
+
+Address the reader directly rather than imagining a hypothetical third party.
+
+{% Compare 'worse', 'Don’t' %}
+> A seller can choose a variety of shopfront themes.
+
+If the reader isn't a seller, this detail may not seem relevant.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> You can choose a variety of shopfront themes.
+
+Using second person helps convey that we think this information is relevant to readers. (This strategy assumes that the post has been appropriately targeted to an [audience](/handbook/audience).)
+{% endCompare %}
+
+Provide a way to measure success. How can developers test that they've followed your instruction correctly?
+
+## Keep it simple
+Focus on essential details when explaining a tool or technique. Avoid overwhelming the reader with "real-world" examples that include complexities that aren't required to understand the basic concept. Save those details for supplemental codelabs or follow-up posts. See the "[Code splitting with React.lazy and Suspense](/code-splitting-suspense)" post for an example.
+
+Avoid jargon. If you absolutely _must_ use a term most web.dev readers are unlikely to know, define it in a short aside. See the "[web.dev components](/handbook/web-dev-components#asides)" post for how to include asides. Also check out the "[Inclusion and accessibility](/handbook/inclusion-and-accessibility#use-readable-language)" post for more info about writing readable language.
+
+{% Compare 'worse', 'Don’t' %}
+> In some scripts, graphemes can be visually joined when they're adjacent.
+
+_Script_ and _grapheme_ are very precise, but they're likely to be unfamiliar to most readers and may cause confusion since readers probably more commonly see _script_ referring to a piece of code.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> In some written languages, certain characters can be visually joined when they're adjacent.
+
+While _written language_ and _character_ are slightly less precise, they should be more familiar and less likely to cause misunderstandings.
+{% endCompare %}
+
+## Be accurate
+Make sure your information aligns to current documentation, standards, and best practices. If you're writing about a topic that isn't in your area of expertise (or even if it is!), get input from other experts.
+
+## Give useful context
+Tell readers what you're expecting them to know up front—and provide links to where they can learn it if you can. See the "[Get started: optimize your React app](/get-started-optimize-react/#what-will-you-learn)" post for an example.
```

Comment:
Use a comma before a coordinating conjunction.

You should ruthlessly purge the style guide of non-standard ways of doing things. They will be taken as license to do the same in articles.